schema.graphql

```diff
@@ -176,6 +176,23 @@ type NominationSpend @entity(immutable: true) {
   transactionHash: Bytes!
 }
 
+enum ActivityType {
+  Vote
+  Stake
+  Ratify
+  Reject
+  Create
+}
+
+type Activity @entity {
+  id: ID!
+  user: User!
+  activityType: ActivityType!
+  timestamp: BigInt!
+  transactionHash: Bytes!
+  proposal: Proposal
+}
+
 type User @entity {
```

Comment:
Should we add a property for Activities in the User entity?

Reply:
Yes, you want to attach it as a relationship and then in activities reference the User.